.vscode/settings.json

```diff
@@ -9,5 +9,6 @@
   "[solidity]": {
     "editor.defaultFormatter": "JuanBlanco.solidity"
   },
-  "coverage-gutters.coverageFileNames": ["lcov.info"]
+  "coverage-gutters.coverageFileNames": ["lcov.info"],
+  "solidity.compileUsingRemoteVersion": "v0.8.21+commit.d9974bed"
```

Comment:
i recommend installing `svm`

Reply:
👍 installed!
I think might still be good to pin a version here since CREATE2 addresses depend on compiler versions.